app/assets/stylesheets/themes/law.scss

```diff
@@ -126,44 +126,68 @@
     }
   }
 
-  .smallMasthead .masthead, .smallMasthead .masthead-old, .bigMasthead .mastheadPlaceholder, .bigMasthead .masthead, .bigMasthead .masthead-old, .nameBar, .rightColumn .styleTwo {
-  background: $law-primary; /* Old browsers */
+  .smallMasthead .masthead,
+  .bigMasthead .mastheadPlaceholder,
+  .bigMasthead .masthead,
+  .nameBar, .rightColumn .styleTwo {
+    background: $law-primary; /* Old browsers */
   }
   
-  .rightColumn .styleTwo .border { background-color: #002663; }
-  .styleTwo p {color: $white;}
-  .styleTwo li {color: $white;}
-  .styleTwo div {color: $white;}
-  .styleTwo h2 {color: $white;}
-  .styleTwo h3 {color: $white;}
-  .rightColumn .styleTwo a {color: $white; border-bottom: 2px dotted #61728d;}
+  .rightColumn .styleTwo .border {
+    background-color: #002663;
+  }
+
+  .styleTwo p {
+    color: $white;
+  }
+
+  .styleTwo li {
+    color: $white;
+  }
+
+  .styleTwo div {
+    color: $white;
+  }
+
+  .styleTwo h2 {
+    color: $white;
+  }
+
+  .styleTwo h3 {
+    color: $white;
+  }
+
```

Comment:
Can we simplify this so all these selectors are combined, or is there a specific reason they're all separated

`.styleTwo p, .styleTwo li,` 
`.styleTwo li,  .styleTwo div,`
`.styleTwo h2, .styleTwo h3 {`
  `color: $white;`
`}`

Reply:
No reason that I can identify. I've reformatted this style block as so...

  .styleTwo {

    p,
    li,
    div,
    h2,
    h3 {
      color: $white;
    }
  }